cmd/checks_windows.go

```diff
@@ -279,6 +291,15 @@ func fileOwner(filePath, owner string) (bool, error) {
 	return theowner == owner, nil
 }
 
+func filePermission(filePath, user, permission string) (bool, error) {
+	cmd := fmt.Sprintf("Get-Acl \"%s\"| Select-Object -ExpandProperty Access | Where-Object identityreference -eq '%s'", filePath, user)
```

Comment:
While it's easier to use posh for these checks, we've been trying to migrate away from relying on it since it tends to be less reliable and lowers the bar for competitors to intercept and modify what a check is doing. If you could explore re-implementing this section through the Windows API (perhaps with the existing dependency iamacarpet/go-win64api?), that would be appreciated.

Reply:
It would also likely eliminate the need for the string handling function `lines2map` below.

Reply:
I replaced checking through Powershell with checking through a system ACL API, and yes, it did remove the need for the lines2map function.